device/alpaka/include/traccc/alpaka/clusterization/clusterization_algorithm.hpp

```diff
@@ -67,12 +71,14 @@ class clusterization_algorithm
         const override;
 
     private:
-    /// The average number of cells in each partition
-    config_type m_config;
     /// The memory resource(s) to use
     traccc::memory_resource m_mr;
     /// The copy object to use
     std::reference_wrapper<vecmem::copy> m_copy;
+    /// The Alpaka queue to use
+    std::reference_wrapper<queue> m_queue;
```

Comment:
This is done inconsistently throughout the code... both here and in the CUDA.

I.e. some places use `vecmem::copy&` and `stream&`/`queue&` and other places use `std::reference_wrapper`.

I don't think this is an "issue" per se, but maybe something we should standardise eventually.